apps/showcase/src/index.html

```diff
@@ -9,5 +9,6 @@
 </head>
 <body>
   <app-root></app-root>
+  <script src ="coi-serviceworker.js"></script>
```

Comment:
should not be part of the `scripts` option of the builder instead?

Reply:
updated